app/controllers/assignments_controller.rb

```diff
@@ -25,10 +25,15 @@ def create
   end
 
   def destroy
-    @assignment.notify :owner, of: :deleted_assignment, by: @current_user
-    @assignment.destroy
-    confirm_change(@assignment)
-    redirect_to roster_assignments_path(@roster)
+    if @current_user.admin_in?(@roster)
+      @assignment.notify :owner, of: :deleted_assignment, by: @current_user
+      @assignment.destroy
+      confirm_change(@assignment)
+      redirect_to roster_assignments_path(@roster)
+    else
+      flash[:errors] = 'Only roster admins may delete assignments.'
```

Comment:
I know that @benmelz  has expressed the opinion of wanting to start using locale based text in round-three, I would assume that should apply here as well?

Reply:
If rubocop was updated I would have seen this!